core/jsonparse.mly

```diff
@@ -11,12 +11,6 @@ struct
 end
 
 let internal_error message =
-  raise (Errors.internal_error ~filename:"jsonparse.mly" ~message)
```

Comment:
I think this line should stay?

Reply:
...whoops, seems I was a little eager with my delete button. Thanks, fixed.